apps/challenges/aws_utils.py

```diff
@@ -95,6 +95,96 @@
 }
 
 
+def scale_resources(challenge, new_cores, new_memory):
+    client = get_boto3_client("ecs", aws_keys)
```

Comment:
@matthew-so  take a look at other fargate worker util methods, for example, `scale_workers` method [here](https://github.com/Cloud-CV/EvalAI/blob/master/apps/challenges/aws_utils.py#L564). Please follow similar docstring and first set of checks. We only want to be able to call the API in prod and staging and not locally.